app/src/components/FilterItem/FilterItem.scss

```diff
@@ -0,0 +1,22 @@
+@import "../../shared/variables";
+
+.filter_item {
```

Comment:
If you use CSS Modules, there are not necessary to use BEM, check https://css-tricks.com/css-modules-part-1-need/#article-header-id-4

Reply:
Done